src/settings/LocationLocations/LocationManager.js

```diff
@@ -310,7 +310,7 @@ class LocationManager extends React.Component {
 
   validate = values => {
     const errors = {};
-    const requiredFields = ['name', 'code', 'discoveryDisplayName', 'institutionId', 'campusId', 'libraryId', 'isActive'];
+    const requiredFields = ['name', 'code', 'discoveryDisplayName', 'institutionId', 'campusId', 'libraryId'];
```

Comment:
When the value for the `isActive` field is `inactive`, the `validate` method recognizes it as the boolean `false` and sets the error `missing a required field`, which is incorrect behavior.
Moreover, since the `Select Status` option is not provided in the` isActive` field, there is no way to set its value as `undefined`.